blocks/header/header.js

```diff
@@ -21,49 +21,47 @@ export default async function decorate(block) {
   block.textContent = '';
 
   // fetch nav content
-  const navPath = cfg.nav || '/nav';
+  const navPath = cfg.nav || '/nav-new';
   const resp = await fetch(`${navPath}.plain.html`);
-  const html = await resp.text();
+  if (resp.status === 200) {
```

Comment:
```suggestion
  if (resp.ok) {
```
Might also include `304`